frontend/packages/ux-editor/src/components/UnsupportedVersionMessage/UnsupportedVersionMessage.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import type { UnsupportedVersionMessageProps } from './UnsupportedVersionMessage';
+import { UnsupportedVersionMessage } from './UnsupportedVersionMessage';
+import { textMock } from '../../../../../testing/mocks/i18nMock';
+
+describe('UnsupportedVersionMessage', () => {
+  it('should render without crashing', () => {
+    const { baseElement } = renderUnsupportedVersionMessage();
+    expect(baseElement).toBeTruthy();
+  });
+
+  it('should render the correct title', () => {
+    renderUnsupportedVersionMessage();
+    expect(
+      screen.getByText(
+        textMock('ux_editor.unsupported_version_message_title', { version: '2.0.0' }),
+      ),
+    ).toBeInTheDocument();
```

Comment:
You don't need to change it, but in cases where we are using `getByRole`, we do not need to use `.toBeInTheDocument()`, since the query it self will fail if it does not exist. If we were using `queryByRole`, then we need to use the `.toBeInTheDocument()`. 😊

Reply:
Assume thats also the case for `getByText` then? 😊 
IMO it's more readable as a test with the `.toBeInTheDocument()` even if it isn't strictly necessary.

Reply:
True, let's keep it with toBeInTheDocument. 😀